### ISSUE TITLE
Unlabeled checkbox when editing profiles.

### ISSUE THREAD

**Issue by rkingett** [liberapay/liberapay.com#1776]
When editing profiles and accounts, there is an unlabeled checkbox just before the username. I'm not quite sure what it does. I checked it, but nothing seemed to happen. The "profile" checkbox below the username is labeled, but perhaps text should be appended that tells screen reader users a menu opens upon checking this box.
